lib/keyrod/oidc_client.rb

```diff
@@ -0,0 +1,44 @@
+require 'faraday'
+require 'json'
+
+module Keyrod
+  class OIDCClient
+    attr_reader :oidc_site, :refresh_token, :client_id, :client_secret, :post_params, :ssl
```

Comment:
Do you really need `post_params` and `ssl` object attributes? I would suggest renaming `set_params` and `set_ssl` methods to `prepare_params` and `prepare_ssl` or just `params` and `ssl`, make them return the values and call these methods directly from `access_token` method without storing the values in the object attributes. What do you think?

Reply:
I tried that in the first place, however it raised AbcSize metric over 20.

Reply:
That is not possible because it won't change `access_token` method in any way.